src/verify_cert.rs

```diff
@@ -185,6 +188,15 @@ fn check_validity(input: &mut untrusted::Reader, time: time::Timespec)
     Ok(())
 }
 
+pub fn parse_validity(validity: &untrusted::Input)
+                  -> Result<(time::Timespec, time::Timespec), Error> {
```

Comment:
Instead of using a tuple, let's create a `struct Validity { notBefore: Timespec, notAfter: Timespec }`. I'm trying to get away from using tuples like this in *ring* and webpki, in general.